astroplan/tests/test_scheduling.py

```diff
@@ -21,70 +22,182 @@
                                      dec=89.26410897 * u.deg), name="Polaris")
 
 apo = Observer.at_site('apo')
-targets = [vega, rigel, polaris]
+targets = [vega, polaris, rigel]
+default_time = Time('2016-02-06 03:00:00')
```

Comment:
It's probably better to move the time into each test, because ideally in the future that will be updated to test multiple times (having only one test time is a pretty narrow test).